presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java

```diff
@@ -37,21 +40,27 @@
 @Path("/v1/info")
 public class ServerInfoResource
 {
-    private final ServerInfo serverInfo;
+    private final NodeVersion version;
+    private final String environment;
+    private final boolean isCoordinator;
```

Comment:
Per standard naming conventions, this should be `coordinator`. The getter would be `isCoordinator`.